### PR TITLE
UI improvements for subtask 2, 6 and 7

### PR DESCRIPTION
**Pull request by blackeagle0188**
2. Showing plot 
- [x] Showing 7k plots on left side list and mpa
- [x] Showing popup for selected plot
- [x] Highlight selected plot on map
6. Search Functionality
- [x] Add filter text and remove filter button
- [x] Test filter function for 7k plots
7. Wallet connect message
- [x] Replace message text

- [x] Add map click function to display plot details and popup
- [x] Don't change zoom level when select plot(Dev task 5)
- [x] Remove green bar(Dev task 6)

### CODE REVIEW

File: packages/react-app/src/components/PlotButton.tsx [hunk 1]
```diff
@@ -9,7 +10,7 @@ import LAND_IMG from "../assets/images/SampleLandImage.png";
 import { stringifyPlotId } from "../helpers/stringifyPlotId";
 
 interface Props {
-  plot: Plot;
+  plot: NewPlot;
```
[the remark this answers is not in the document]
Comment (blackeagle0188) [reply packages/react-app/src/components/PlotButton.tsx:13]: Got it, Will do it.

File: packages/react-app/src/components/PlotMap.jsx [hunk 2]
```diff
@@ -82,33 +118,66 @@ export default function PlotMap({ parcel, plots, startingCoordinates, startingZo
...
-        }, 1000);
-      });
-    }
-  }, [map.current, plots]);
+  // useEffect(() => {
+  //   if (map?.current) {
+  //     map.current.on("load", function () {
+  //       if (map.current && map.current.getSource("parcel")) return; // skip if already added
+  //       addOutlineToMap(parcel.metadata.geojson, "parcel");
+  //       communal.forEach((plot, idx) => {
```
[the remark this answers is not in the document]
Comment (blackeagle0188) [reply packages/react-app/src/components/PlotMap.jsx:126]: This code was used to draw highlight when hover on plot in left side(original feature)

**Review by ssdsr — COMMENTED** [review]
Looks great 🙌

File: packages/react-app/src/components/ProgressBar.tsx [hunk 3]
```diff
@@ -3,11 +3,12 @@ import { useAppSelector } from "../hooks";
 
 export default function ProgressBar() {
```
[the remark this answers is not in the document]
Comment (blackeagle0188) [reply packages/react-app/src/components/ProgressBar.tsx:4]: removed it

File: packages/react-app/src/views/BrowsePlots.tsx [hunk 4]
```diff
@@ -44,7 +46,8 @@ export default function BrowsePlots({ networkProvider, web3Modal }: Props) {
       });
     } else {
       toast.dismiss("isWhitelisted");
-      toast.error("You aren't whitelisted to buy any plots yet 😢", {
+      console.log(web3Modal)
```
Comment (trkaplan) [inline packages/react-app/src/views/BrowsePlots.tsx:49]: console.log left. But not important

File: packages/react-app/src/models/Plot.ts [hunk 5]
```diff
@@ -1,15 +1,22 @@
-export interface Plot {
-  id: number;
-  parcel: number;
-  sold: boolean;
-  owner: string;
-  price?: string;
-  metadata: PlotMetadata;
-}
+// export interface Plot {
```
Comment (trkaplan) [inline packages/react-app/src/models/Plot.ts:1]: You can feel free to remove commented code blocks.
If we need to go back, we can check the commit history or check this branch: https://github.com/citydaoproject/app/tree/old-version-with-minting-feature (I had created a branch before we started working on improvements)

**Review by trkaplan — DISMISSED** [review]

**Review by trkaplan — APPROVED** [review]

**Review by trkaplan — APPROVED** [review]

**Pull request merged**